app/javascript/elm/src/Main.elm

```diff
@@ -69,9 +70,14 @@ type alias Model =
     , scrollPosition : Float
     , debouncingCounter : Int
     , isNavExpanded : Bool
+    , isCustomThemeOn : Tagged Theme Bool
```

Comment:
YEEEEES! I LOVE THIS PACKAGE!

Reply:
will refactor `isNavExpanded` same way later